spec/predicates/vuln.md

```diff
@@ -64,41 +64,39 @@ The `predicate` contains a JSON-encoded data with the following fields:
 
 > > > The version of the Vulnerability DB.
 
-**scanner.db.lastUpdate, required** string (timestamp)
+**scanner.db.lastUpdate, optional** string (timestamp)
 
 > > > The timestamp of when the vulnerability DB was updated last time.
 
-**scanner.result, required** list
+**scanner.result, required** object list
 
 > > The result contains a list of vulnerabilities. Note that an empty list means the **scanner** found no vulnerabilities.
 > > This is the most important part of this field because it'll store the scan result as a whole. So, people might want
 > > to use this field to take decisions based on them by making use of Policy Engines tooling whether allow or deny these images.
+> > Each object defines information about each one of the vulnerabilities found by the scanner.
 
-**scanner.result.[*].vulnerability, optional** object
 
-> > > The vulnerability object defines information about each one of the vulnerabilities found by the scanner.
+**scanner.result.[*].id, required** string
```

Comment:
These changes look good to me, we can remove the vulnerability, and assume they refer to vulnerabilities.